releases/24.09.0/24090 Release Notes.md

```diff
@@ -0,0 +1,469 @@
+* Network heartbeat packets are sent during high CPU usage; avoids network disconnection due to timeout.  
+  
+
+
+**Improvements:**
+
+* Added support for PGM Image Format  
+* Added support for templates level creation  
+* Resolved several issues where networked games could ‘timeout’ due to threads being blocked due to loading of a large level.  
+* Allowing non anti aliased fonts in Lua Editor  
+* Improved Lua Editor syntax highlighting and style updates  
+* Many prefab quality of life improvements  
+* Prefab Container entities can now be set as Editor Only. Doing so causes them to be skipped at runtime, with only the contents of the prefab spawned into the level.  
+* Migrate OffsetKeyPosition for AZ::Vector3 for CTrackViewTrack  
+* Improved Asset Processor asset analysis performance  
+* Linux Console logging  
+* Ability to set different max path lengths per platform (resolves a commonly encountered issue when installing O3DE with a deep path).  
+* Improved ability to import models and objects from external sources  
+* Project manager add project dialog to target project.json instead of any directory  
+* Add r\_fogLayerSupport and r\_fogTurbulenceSupport CVARS for quality  
+* Add exponential and exponential squared fog modes  
+* ScriptCanvas support Quaternion CreateFromValues  
+* Improved HDRiSkybox scripting API  
+* Resolved many Script Canvas quality of life issues  
+* Improved editor performance with large number of entities  
+* Added support for ray-traced reflections in SpecularReflectionsFeatureProcessor  
+* Many Asset Browser quality of life improvements  
+* Terrain | Add Help Url to components that have docs pages   
+* MiniAudio: Added more volume controls and a pause method for Playback component  
+* Add quality settings to default project template  
+* Many Atom performance improvements for all supported platforms  
+* Added support for renaming gems  
+* Added Support For Raytracing Against Skinned Meshes  
+* Added functionality for material pipeline files to specify custom object SRG members  
+* Added support for lighting channels to all light types  
+* Added support for configuring anti-aliasing and multi-sample count via CVars  
+* Added support of node-searching in script canvas  
+* Added support to turn on/off directional lights shadow  
+* Added support to run clients headlessly for simulation client  
+* MultiplayerSample gamepad player input support  
+* Added vertex color support to BasePBR, StandardPBR, EnhancedPBR material types as well as material canvas.  
+* [Added three new tone-mapping modes: AcesFitted, AcesFilmic and Filmic](https://github.com/o3de/o3de/commit/af81e8362e5a9a15047d35d34573e4aacceb7941)  
+* [Added shadow support for SimpleSpotLights](https://github.com/o3de/o3de/commit/7ab669f974a6df543339a1cc625c34f6ad21b282)  
+* [Added Gobo (i.e texture projection) support for simple spot light.](https://github.com/o3de/o3de/commit/08e1d396a40b56db24ee26952da935c625d9def1)  
```

Comment:
I would remove the links (or add them everywhere) to keep the list more consistent
```suggestion
* Added three new tone-mapping modes: AcesFitted, AcesFilmic and Filmic
* Added shadow support for SimpleSpotLights
* Added Gobo (i.e texture projection) support for simple spot light.
```